### PR TITLE
refactor: replace 'Command' with 'kclvm-go' when 'kpm run'.

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

issue #85 

#### 2. What is the scope of this PR (e.g. component or file name):

kpm run

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [x] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

replace 'Command' with 'kclvm-go' when 'kpm run'.

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [x] Unit test
- [x] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: pkg/opt/opt.go [hunk 1]
```diff
@@ -175,47 +178,50 @@ func (oci *OciOptions) AddStoragePathSuffix(pathPrefix string) string {
...
 }
 
-// Generate the kcl compile command arguments based on 'KclvmOptions'.
-func (kclOpts *KclvmOptions) Args() []string {
-	var args []string
-	args = append(args, kclOpts.EntryFiles...)
-	args = append(args, kclOpts.PkgPathMapArgs()...)
-	if len(kclOpts.KclvmCliArgs) != 0 {
-		args = append(args, strings.Split(kclOpts.KclvmCliArgs, " ")...)
+func NewKclvmOpts() *KclvmOptions {
```
[the remark this answers is not in the document]
Comment (zong-zhe) [reply pkg/opt/opt.go]: fixed.

File: pkg/opt/opt.go [hunk 2]
```diff
@@ -175,47 +178,50 @@ func (oci *OciOptions) AddStoragePathSuffix(pathPrefix string) string {
...
+func (kclOpts *KclvmOptions) AddDep(depName string, depPath string) {
+	kclOpts.Deps = append(kclOpts.Deps, fmt.Sprintf(EXTERNAL_PKGS_ARG_PATTERN, depName, depPath))
 }
 
-func NewKclvmOpts() *KclvmOptions {
-	return &KclvmOptions{
-		Deps:       make(map[string]string),
-		EntryFiles: make([]string, 0),
+// GetDepOpts will return the dependency options.
+func (kclOpts *KclvmOptions) GetDepOpts() *kcl.Option {
```
[the remark this answers is not in the document]
Comment (zong-zhe) [reply pkg/opt/opt.go]: fixed.

File: pkg/cmd/cmd_run.go [hunk 3]
```diff
@@ -194,28 +194,19 @@ func runPkgInPath(pkgPath, entryFilePath string, vendorMode bool, kclArgs string
...
-	if err != nil {
-		return "", err
-	}
-
-	kclvmCmd, err := runner.NewCompileCmd(compileOpts)
-
-	if err != nil {
-		return "", err
-	}
+	kclvmCompiler := runner.NewCompiler(compileOpts)
```
[the remark this answers is not in the document]
Comment (zong-zhe) [reply pkg/cmd/cmd_run.go]: fixed.

File: pkg/opt/opt.go [hunk 4]
```diff
@@ -175,47 +178,50 @@ func (oci *OciOptions) AddStoragePathSuffix(pathPrefix string) string {
 
 // The parameters needed to compile the kcl program.
 type KclvmOptions struct {
-	Deps         map[string]string
-	EntryFiles   []string
+	Deps         []string
+	EntryFile    string
```
[the remark this answers is not in the document]
Comment (zong-zhe) [reply pkg/opt/opt.go]: kpm currently supports specifying one entry file with the `-input` parameter, and multiple *.k files can be compiled by `kpm run` under the package directory or `kcl.yaml`.

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**